app/kubemci/pkg/gcp/backendservice/backendservicesyncer.go

```diff
@@ -59,7 +59,7 @@ func (b *BackendServiceSyncer) EnsureBackendService(lbName string, ports []ingre
 	var err error
 	ensuredBackendServices := BackendServicesMap{}
 	for _, p := range ports {
-		be, beErr := b.ensureBackendService(lbName, p, hcMap[p.Port], npMap[p.Port], igLinks, forceUpdate)
+		be, beErr := b.ensureBackendService(lbName, p, hcMap[p.NodePort], npMap[p.NodePort], igLinks, forceUpdate)
```

Comment:
This seems unrelated to latest dependencies. Is it intentional? The linked triggers don't clarify this either. 

Could you please explain this change in the PR description or link to the relevant trigger? Same question for all the `Port` -> `NodePort` changes below. I think I am missing context.

Reply:
Added more context. This is because the struct that we are using from ingress-gce renamed Port to NodePort: kubernetes/ingress-gce#128